frontend/components/forms/fields/Checkbox/_styles.scss

```diff
@@ -22,6 +22,12 @@
         border: solid 2px $core-vibrant-blue;
       }
 
+      &--disabled {
+        &::after {
+          @include disabled;
```

Comment:
just want to add the grey `background-color` here. Maybe in the `disabled` mixin if we want that globally?